critiquebrainz/test_config.py

```diff
@@ -4,6 +4,7 @@
 WTF_CSRF_ENABLED = False
 
 SQLALCHEMY_DATABASE_URI = "postgresql://critiquebrainz:critiquebrainz@db_test:5432/critiquebrainz"
+MB_DATABASE_URI = "postgresql://musicbrainz:musicbrainz@musicbrainz_db:5432/musicbrainz_db"
```

Comment:
Please change the database name to `musicbrainz_test` as I had mentioned in my first comment. :)

Reply:
Accidentally passed by that bit of advice, updated now.